### PR TITLE
updated version of ktlint to 9.4.1, trailing passing

### PR DESCRIPTION
**Pull request by tomklecha**
updated version of ktlint

### CODE REVIEW

**Comment by tomklecha** [comment]
there was an issue on older version, but apparently now is fixed
[(https://github.com/pinterest/ktlint/issues/729)]
just don't understand while the operate with two ktlint versions -> ours for gradle plugin and other (0.40.0) for ktlint version??

File: features/home/build.gradle [hunk 1]
```diff
@@ -44,5 +44,5 @@ dependencies {
     implementation "androidx.test.ext:junit:${Versions.junitAndroid}"
     implementation "androidx.test.espresso:espresso-core:${Versions.espressoAndroid}"
     implementation "androidx.navigation:navigation-testing:${Versions.navigationComponent}"
-    implementation "androidx.fragment:fragment-testing:${Versions.fragmentTest}"
+    debugImplementation "androidx.fragment:fragment-testing:${Versions.fragmentTest}"
```
Comment (Canato) [inline features/home/build.gradle:47]: why this change?

**Review by Canato — APPROVED** [review]
Nice PR @tomklecha 

I didn't understood your question
> just don't understand while the operate with two ktlint versions -> ours for gradle plugin and other (0.40.0) for ktlint version??

File: features/home/build.gradle (hunk 1 above)
Comment (tomklecha) [reply features/home/build.gradle:47]: android studio red alert in lint to replace it, also it was breaking the pipeline

Comment (tomklecha) [reply features/home/build.gradle:47]: the question is about the plugin version.
Because our plugin version is 9.4.1. But in the page they speak about version 0.40.0.
I just do not understand the difference

Comment (Canato) [reply features/home/build.gradle:47]: Since we are here, let's improve our gradles. For this case and all other test libraries let's use `testImplementation` =D

`testImplementation "androidx.fragment:fragment-testing:${Versions.fragmentTest}"`

**Comment by Canato** [comment]
> the question is about the plugin version.
Because our plugin version is 9.4.1. But in the page they speak about version 0.40.0.
I just do not understand the difference

Ahh ok! so we use this library https://github.com/JLLeitschuh/ktlint-gradle (version 9.4.1)
This library allow us to use ktlint (version 0.40.0)
> Provides a convenient wrapper plugin over the ktlint project.

File: features/home/build.gradle (hunk 1 above)
Comment (Canato) [reply features/home/build.gradle:47]: after change this, can merge @tomklecha

Comment (tomklecha) [reply features/home/build.gradle:47]: I cannot change it to testImplementation,
[https://developer.android.com/jetpack/androidx/releases/fragment#declaring_dependencies] even here you have it implemented as debug.
So I can change rest to testImplementation and then merge it

**Comment by tomklecha** [comment]
since we do UI test in home fragment - we must use androidTestImplementation

**Review by Canato — APPROVED** [review]
